src/game.cpp

```diff
@@ -9111,6 +9111,57 @@ void add_disassemblables( uilist &menu, map_stack &items,
     }
 }
 
+// Butchery sub-menu and time calculation
+void butcher_submenu( map_stack &items, const std::vector<int> &corpses, int corpse = -1 )
```

Comment:
This function needs its indentation corrected, the build bots are letting you get away with it because it's in game.cpp, which hasn't had it's styling completed.

Reply:
Running that function (or the entire document) through astyle doesn't change anything. What's wrong with the indentation?

Reply:
My mistake, it displayed as un-indented on my phone, no idea why it did that.